tests/cephfs/cephfs_tier1_ops.py

```diff
@@ -0,0 +1,408 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Pre-requisites :
+    1. Requires 2 client nodes in the setup
+
+    Operations performed :
+    1. Enable Multiple File systems 1. In Replicated 2. In EC
+    2. Create 2 SubVolumeGroups on each file system
+    3. Create 2 Sub volumes on each of the subvolume group Size 20 GB
+    4. Create 2 sub volumes on default subvolume group
+    5. Mount 1 subvolumegroup/subvolume on kernal and 1 subvloume on Fuse → Client1
+    6. Mount 1 subvolumeon kernal and 1 subvloumegroup/subvolume on Fuse → Client2
+    7. On EC,Mount 1 subvolumegroup/subvolume on kernal and 1 subvloume on Fuse → Client2
+    8. On EC,Mount 1 subvolumeon kernal and 1 subvloumegroup/subvolume on Fuse → Client1
+    9. Run IOs on subvolumegroup/subvolume on kernel client and subvolume in Fuse clientusing below commands
+        git clone https://github.com/distributed-system-analysis/smallfile.git
+        cd smallfile
+        for i in create read append read delete create overwrite rename delete-renamed mkdir rmdir create symlink
+        stat chmod ls-l delete cleanup  ;
+        do python3 smallfile_cli.py --operation $i --threads 8 --file-size 10240 --files 100 --top /mnt/kcephfs/vol5/
+        ; done
+        IO Tool 2 :
+        wget -O linux.tar.gz http://download.ceph.com/qa/linux-5.4.tar.gz
+        tar -xzf linux.tar.gz tardir/ ; sleep 10 ; rm -rf  tardir/ ; sleep 10 ; done
+        DD on Each volume:
+
+        Wget :
+       http://download.eng.bos.redhat.com/fedora/linux/releases/34/Server/x86_64/iso/Fedora-Server-dvd-x86_64-34-1.2.iso
+       http://download.eng.bos.redhat.com/fedora/linux/releases/33/Server/x86_64/iso/Fedora-Server-dvd-x86_64-33-1.2.iso
+       http://download.eng.bos.redhat.com/fedora/linux/releases/32/Server/x86_64/iso/Fedora-Server-dvd-x86_64-32-1.6.iso
+        Note : Run 2 IO tools mentioned above on each volume mounted
+    10. Create Snapshots.Verify the snap ls | grep to see snap ot created
+    11. Create Clones on all 8 volumes. Verify the clones got created using clone ls(subvolume ls)
+    12. Set File Level Quota on 2 directories under subvolume and Size Level Quota on under 2 directories
+        under subvolume.
+    13. Verify quota based on your configuration
+    14. Clear Quotas
+    15. Remove Clones
+    16. Remove Snapshots
+    17. Unmount
+    18. Remove Volumes.
+
+    Args:
+        ceph_cluster:
+        **kw:
+
+    Returns:
+
+    """
+    try:
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        results = []
```

Comment:
Not sure why we would need to have `results`. Instead of holding it in memory... we could write an internal method that wraps around `exec_command` so that you could log the stdin/stdout,stderr every time... 

The wrapper would create a lot of noise that would make triage difficult